src/app.js

```diff
@@ -36,6 +45,20 @@ app.use((req, res, next) => {
   next();
 });
 
+passport.use('local', new LocalStrategy(
+  {
+    session: false,
+  },
+  (username, password, done) => {
```

Comment:
instead of this:
```javascript
(username, password, done) => {
  if (testUser.userName !== username
    && testUser.password !== password) {
    done(null, false, 'bad username');
  } else {
    done(null, testUser);
  }
},
```
you can use this:
```javascript
(username, password, done) =>
  (testUser.userName !== username
    && testUser.password !== password)
    ? done(null, false, 'bad username');
    : done(null, testUser);
```
function without body is usually cleaner and more safe: since it return result of expression instead of having body with statements, it never has local variables; if you don't have variable you don't have mutable state; if you don't have mutable state function, function will be much safer to use and even reuse in other places.